src/experimental/test/property/arbitraries/canister_methods/post_upgrade_method_arb.ts

```diff
@@ -101,28 +81,19 @@ export function PostUpgradeMethodArb<
                     namedParams,
                     returnType,
                     generator.generateBody,
-                    methodImplementationLocation,
-                    functionName,
-                    methodName,
-                    api
+                    functionName
                 );
 
                 const variableAliasDeclarations = paramTypes
                     .flatMap((param) => param.src.variableAliasDeclarations)
                     .filter(isDefined);
 
-                const globalDeclarations =
-                    methodImplementationLocation === 'STANDALONE'
-                        ? [...variableAliasDeclarations, methodImplementation]
-                        : variableAliasDeclarations;
+                const globalDeclarations = variableAliasDeclarations;
```

Comment:
Why does `postUpgrade` have `globalDeclarations` but `inspectMessage` doesn't?

Reply:
post upgrade has parameters whose types might be globally defined